jupyter-docker/spark-defaults.conf.template

```diff
@@ -26,4 +16,16 @@ spark.executor.extraClassPath ./gcs-connector.jar:./hail-0.1-[HAILHASH]-Spark-2.
 # Hail needs at least 50GB
 
 spark.sql.files.maxPartitionBytes=100000000000
-spark.sql.files.openCostInBytes=100000000000
\ No newline at end of file
+spark.sql.files.openCostInBytes=100000000000
+
+###################
+# A-Team additions
```

Comment:
Maybe in the future we can let the user specify spark configs, like how the Google Dataproc API supports `SoftwareConfig`:
https://cloud.google.com/dataproc/docs/reference/rest/v1beta2/ClusterConfig#SoftwareConfig

But it makes sense for us to specify reasonable defaults.